webapp/src/components/show_secret_modal.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import {LegalHold} from '@/types';
+import {GenericModal} from '@/components/mattermost-webapp/generic_modal/generic_modal';
+
+import './create_legal_hold_form.scss';
+
+interface ShowSecretModalProps {
+    legalHold: LegalHold | null;
+    onExited: () => void;
+    visible: boolean;
+}
+
+const ShowSecretModal = (props: ShowSecretModalProps) => {
+    const onCancel = () => {
+        props.onExited();
+    };
+
+    const createMessage = (lh: LegalHold|null) => {
+        if (lh) {
+            return (
+                <React.Fragment>
+                    <div
+                        style={{
+                            textAlign: 'center',
+                        }}
+                    >
+                        <p>{'The secret key to check the authenticity of the legal hold is: '}</p>
```

Comment:
It looks like the permissions to view the secret are based on only sys admins being able to get to the system console.  This is fine for now, but in a future version we'll have to check here that they have permissions (sys admin or future legal hold admin role) and we should probably allow different users to create legal holds such they only they can see the secret.   We'll get some customer feedback first.